style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+}
+
+nav {
+  width: 100%;
+  height: 50px;
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+nav a {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  text-decoration: none;
+  font-size: 18px;
+}
+
+.nav-items {
+  display: none;
+  background-color: #6a6765;
+}
+
+.nav-items ul {
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-items li {
+  list-style: none;
+  padding: 10px 24px;
+}
+
+.nav-items li a {
+  list-style: none;
+  color: #fff;
+  text-decoration: none;
+  font-family: 'Roboto', sans-serif;
+}
+
+/* Home */
+.home {
+  height: 590px;
+  color: #fff;
+  padding: 100px 24px;
+  background: #1c1a19 url("resources/Group.png") 100% 110% no-repeat;
+  background-size: 120px;
+}
```

Comment:
- [ ] Dear @popey17, according to the main requirement we are expected to have this section aligned with `flex-box`. This gives us a lot of options to align all our elements, plus, it's responsive.